src/main/java/pl/craftserve/radiation/RadiationPlugin.java

```diff
@@ -72,13 +74,54 @@ public void onEnable() {
         // Enabling
         //
 
+
         RegionContainer regionContainer = WorldGuard.getInstance().getPlatform().getRegionContainer();
 
+
         for (String worldName : worldNames) {
             if (regionName == null) {
                 break;
             }
 
+            if(config.getBoolean("use-radius")){ //check if user wants to use radius instead of handmade region
+                int radius = config.getInt("radius");
+
+                World world = server.getWorld(worldName);
+
+                Location spawnLocation = world.getSpawnLocation();
+
+                BlockVector3 origin = BlockVector3.at(spawnLocation.getX(), spawnLocation.getY(), spawnLocation.getZ());
+                BlockVector3 size = BlockVector3.ONE.multiply(radius);
```

Comment:
Problem w tym, że to będzie się robiło na wszystkich osiach, także Y, czyli wysokość. Wydaje mi się, że wysokość powinna być tutaj zawsze na pełną wysokość świata.